src/components/MessageInput/UploadProgressIndicator.tsx

```diff
@@ -1,12 +1,28 @@
-import React from 'react';
-import { ActivityIndicator, Image, View } from 'react-native';
-import PropTypes from 'prop-types';
-import styled from 'styled-components/native';
+import React, { PropsWithChildren } from 'react';
+import {
+  ActivityIndicator,
+  GestureResponderEvent,
+  Image,
+  ImageSourcePropType,
+  View,
+} from 'react-native';
 
-import iconReload from '../../images/reload1.png';
+import { styled } from '../../styles/styledComponents';
 import { themed } from '../../styles/theme';
 import { ProgressIndicatorTypes } from '../../utils/utils';
 
+const iconReload: ImageSourcePropType = require('../../images/reload1.png');
```

Comment:
been using `ImageRequireSource` for the other icons, should figure out which is more appropriate